spec/rails_helper.rb

```diff
@@ -8,3 +8,19 @@
 def jsonapi_response
   JSON.parse(response.body, symbolize_names: true)
 end
+
+def post_with_rails_fix(url, params: {}, **others)
```

Comment:
Love this, but perhaps some comment that would wrap this method would be useful for future developers to come and maintain?